ouster_client/include/ouster/client.h

```diff
@@ -97,6 +97,30 @@ std::shared_ptr<client> init_client(const std::string& hostname,
                                     timestamp_mode ts_mode = TIME_FROM_UNSPEC,
                                     int lidar_port = 0, int imu_port = 0,
                                     int timeout_sec = 60);
+
+/**
+ * [BETA] Connect to and configure the sensor and start listening for data via
+ * multicast.
```

Comment:
can we specify here that currently this is supported/tested only on Linux (though it will build on Mac/Windows)

Reply:
Discussed with @Samahu - we will test it later on our side. Since it's marked [BETA], we think it's covered for now.
Will resolve conversation.